core/arch/arm/plat-sunxi/conf.mk

```diff
@@ -0,0 +1,40 @@
+PLATFORM_FLAVOR ?= bpi_zero
+
+sun8i_h2_plus_flavorlist = bpi_zero
+
+ifneq (,$(filter $(PLATFORM_FLAVOR),$(sun8i_h2_plus_flavorlist)))
+include core/arch/arm/cpu/cortex-a7.mk
+$(call force,CFG_SUN8I_H2_PLUS,y)
+endif
+
+ta-targets = ta_arm32
+
+ifeq ($(filter y,$(CFG_SUN8I_H2_PLUS)),y)
+$(call force,CFG_ARM32_core,y)
+$(call force,CFG_GENERIC_BOOT,y)
+$(call force,CFG_GIC,y)
+$(call force,CFG_SUNXI_UART,y)
+$(call force,CFG_PM_STUBS,y)
+$(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
+$(call force,CFG_WITH_LPAE,n)
+$(call force,CFG_WITH_PAGER,n)
+endif
+
+ifneq (,$(filter $(PLATFORM_FLAVOR),bpi_zero))
+CFG_CRYPTO_SIZE_OPTIMIZATION ?= n
+CFG_NUM_THREADS ?= 4
+CFG_TEE_CORE_NB_CORE ?= 4
+CFG_WITH_STACK_CANARIES ?= y
+CFG_BOOT_SECONDARY_REQUEST ?= y
+CFG_PSCI_ARM32 ?= y
+CFG_WITH_STACK_CANARIES ?= y
+CFG_WITH_STATS ?= y
+CFG_NS_ENTRY_ADDR ?= 0x42000000
+CFG_DT ?= y
+CFG_INIT_CNTVOFF ?= y
+CFG_SECONDARY_INIT_CNTFRQ ?= y
+CFG_TZDRAM_START ?= 0x5c000000
+CFG_TZDRAM_SIZE ?= 0x03e00000
+CFG_SHMEM_START ?= 0x5fe00000
+CFG_SHMEM_SIZE ?= 0x00200000
+endif
```

Comment:
I think there are too many conditionals in this file given that only one board is currently supported! (`bpi_zero`)...

Personally I would just drop all the `ifneq` and `ifeq` and also drop `sun8i_h2_plus_flavorlist` and `CFG_SUN8I_H2_PLUS`, just keep `PLATFORM_FLAVOR ?= bpi_zero` (although not strictly needed) for documentation purposes essentially. Then we would see what's needed if/when new `plat-sunxi` boards are supported.

That being said if you prefer to keep the code as is, I'm fine with that.